Kernel/Syscall.cpp

```diff
@@ -162,6 +165,24 @@ ErrorOr<FlatPtr> handle(RegisterState& regs, FlatPtr function, FlatPtr arg1, Fla
 
 }
 
+static const Array<u16, Syscall::Function::__Count> s_syscall_notes {
```

Comment:
Maybe leave the formula you used somewhere

Reply:
Good point. I added a description of where these numbers come from, why I chose *these* numbers, and how to extend them. This will break when the next person adds or removes a syscall, but hopefully the comment makes it clear how to add/remove a number.

Reply:
> This will break when the next person adds or removes a syscall

Didn't we have some kind of static assert that allows for making sure that the table doesn't get out of sync?

Reply:
That's already built into the Array itself. See the `Syscall::Function::__Count` template parameter? Array will complain if it doesn't receive exactly that amount.